integration-cli/docker_cli_external_graphdriver_unix_test.go

```diff
@@ -349,12 +349,12 @@ func (s *DockerExternalGraphdriverSuite) TestExternalGraphDriver(c *check.C) {
 }
 
 func (s *DockerExternalGraphdriverSuite) testExternalGraphDriver(name string, ext string, c *check.C) {
-	s.d.StartWithBusybox(c, "-s", name)
+	s.d.StartWithBusybox(c, "-s", name, "--experimental")
```

Comment:
Actually, `setupTest` already sets up an `ExperimentalDaemon`. So this should not be necessary.